code/modules/busy_space/organizations.dm

```diff
@@ -435,6 +435,7 @@
 	destination_names = list(
 						"Venus",
 						"Earth",
+						"Mercury",
```

Comment:
//VOREStation Edit (or remove this edit since you copied the stuff in this file anyway)

Reply:
Ah crap not again.